### DISCUSSION TITLE
What is the optimal method for serializing or saving imported STEP objects?

### DISCUSSION THREAD

**Discussion by msngupta** [sgenoud/replicad#150] [How To]
Hi,
I am trying to import .stp files, which takes some time. Once they are loaded, I want to serialize the loaded object. However, JSON.stringify doesn't seem to work. Is there a native format to which I can save the files and then import them at faster rates?

Thanks

**Comment by sgenoud** [comment]
Unfortunately there is nothing like that exposed. There might be an opencascade format but you would need to play with a custom ocjs build (and wrap it correctly).

Can you share what the STEP file looks like?
